packages/webapp/public/locales/en/expense.json

```
@@ -1,16 +1,54 @@
 {
-  "EQUIPMENT": "Equipment",
-  "FERTILIZER": "Fertilizer",
-  "SOIL_AMENDMENT": "Soil Amendment",
-  "MACHINERY": "Machinery",
-  "PEST_CONTROL": "Pest Control",
-  "FUEL": "Fuel",
-  "LAND": "Land",
-  "SEEDS_AND_PLANTS": "Seeds and Plants",
-  "MISCELLANEOUS": "Miscellaneous",
-  "UTILITIES": "Utilities",
-  "LABOUR": "Labour",
-  "INFRASTRUCTURE": "Infrastructure",
-  "TRANSPORTATION": "Transportation",
-  "SERVICES": "Services"
+  "EQUIPMENT": {
+    "EXPENSE_NAME": "Equipment",
+    "CUSTOM_DESCRIPTION": "Expenses related to simple tools, supplies, and parts for operating your farm."
+  },
+  "SOIL_AMENDMENT": {
+    "EXPENSE_NAME": "Soil Amendment",
+    "CUSTOM_DESCRIPTION": "Expenses related to additives such as fertilizers that improve the attributes of your soil."
+  },
+  "MACHINERY": {
+    "EXPENSE_NAME": "Machinery",
+    "CUSTOM_DESCRIPTION": "Expenses related to the purchase, lease, rental, and maintenance of machinery."
+  },
+  "PEST_CONTROL": {
+    "EXPENSE_NAME": "Pest Control",
+    "CUSTOM_DESCRIPTION": "Expenses related to the purchase of ingredients, products, and devices used to  manage unwanted species."
```

```suggestion
    "CUSTOM_DESCRIPTION": "Expenses related to the purchase of ingredients, products, and devices used to manage unwanted species."
```

Just an extra space 😅